<?xml version="1.0" ?>
<sdf version='1.5'>
  <model name='iris_rplidar'>

    <include>
      <uri>model://iris</uri>
    </include>

    <include>
      <uri>model://rplidar</uri>
      <pose>0 0 0.1 0 0 0</pose>
    </include>
    <joint name="lidar_joint" type="fixed">
      <child>rplidar::link</child>
      <parent>iris::base_link</parent>
    </joint>

  </model>
</sdf>
<!-- vim: set noet fenc=utf-8 ff=unix sts=0 sw=4 ts=4 : -->

